fast_ignore.gemspec

```diff
@@ -22,11 +23,7 @@ Gem::Specification.new do |spec|
     spec.metadata['changelog_uri'] = 'https://github.com/robotdana/fast_ignore/blob/master/CHANGELOG.md'
   end
 
-  # Specify which files should be added to the gem when it is released.
-  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  end
+  spec.files = FastIgnore.new(include_rules: ['bin/*', 'lib/**/*', 'LICENSE.txt', 'README.md'], relative: true).sort
```

Comment:
Hi, thanks for this, I think i prefer to include CHANGELOG.md here too, and exclude bin/*. 

bin/* items are for development of this gem not for users of this gem. and CHANGELOG.md feels like documentation that should be included for users of the gem. 

Also `lib/**/*` is equivalent to `lib`: the include_rules also include all children, just as ignore rules ignore all children. 

Thank you

Reply:
Ah, I should've know; apologies!
These are fixed now :smile:

Reply:
Tests have failed because my spellcheck linter thinks README is a spelling error, 
Please run `spellr -i` and add `readme` to the english wordlist.

Reply:
Oops. Interesting and fixed!